the-new-developers/gatehopper
# Added a "Return" button to the "Game Over" screen

## Changes:
Closes #23
- Created a "Return" button and added it to the "Game Over" screen. It will take the player to the "Main Menu" if they click it.

![Game Over Screen](https://user-images.githubusercontent.com/79951274/129999024-57020499-b5a9-493d-989f-523139bc2070.JPG)

## How to test this PR:
- Change "currentScene" in the index.js file from "Main Menu" to "Game Over".
- Run the index.html file. The "Game Over" screen will appear instead of the "Main Menu".
- Click the "Return" button. You will be taken to the "Main Menu".

## Checklist:
